### DISCUSSION TITLE
Pogrammatic scrolling of the graph to start at a particular data point

### DISCUSSION THREAD

**Discussion by Praveen-Android** [patrykandpatrick/vico#869] [Questions] [answered]
### Question

Hello, 

Is it possible to programmatically scroll/move the graph to a target data point like on a next/previous button click. I am learning this library and any hints on how to do it would be great.

Thanks!

### Vico version(s)

2.0.0-alpha.28

**Comment by Gowsky** [comment, marked as answer]
Hello @Praveen-Android. Please see [the wiki](https://patrykandpatrick.com/vico/wiki/cartesian-charts/hosts#scroll-and-zoom).

**Reply by Praveen-Android** [reply]
Appreciate the reference!
